conf/global.json

```diff
@@ -19,6 +19,10 @@
     },
     "monitoring": {
       "create_sns_topic": true
+    },
+    "rules_table": {
+      "read_capacity": 20,
+      "write_capacity": 2
```

Comment:
My personal preference - let's not go below 5 for capacity. It's the default, it's cheap, and it works well for most simple use cases. Of course, we should enable autoscaling eventually, but for now 5 seems fine with some wiggle room

Reply:
okay good suggestion - I had set it at 2 because this will be written to fairly rarely, but will set to 5 at minimum